### DISCUSSION TITLE
How to create/add a tag/field matching an IP parameter

### DISCUSSION THREAD

**Discussion by simoneBIP** [Security-Onion-Solutions/securityonion#7951] [Unsupported Versions]
Good afternoon everybody,
I would like to add/create a new tag/field for each log based on the value of another field such as IP source or IP destination in order to match that IP with the corresponding Network/subnet address.
For example: 
Suppose that I have 3 network: Room1: 192.168.1.0/24 , Room2: 192.168.2.0/24 , Room3: 192.168.3.0/24, 
I would like to add a field to each of my log named for example "subnet" which assumes the value of one of these networks based on destination.IP.   
If we assume that my log has destination.IP= 192.168.1.2 then the new field will have subnet=Room1

Do you have any suggestion on how to implement this? 

Thanks in advance,
Simone

**Comment by dougburks** [comment]
You should be able to do something like that via custom Elasticsearch ingest pipeline:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing
